packages/textkit/src/engines/linebreaker/index.js

```diff
@@ -17,6 +18,48 @@ const opts = {
   shrink: 9,
 };
 
+/**
+ * Get regions of urls in the attributed string
+ *
+ * @param {Object} attributedString attributed string
+ * @returns {Array} regions of urls
+ */
+const getUrlRegions = attributedString => {
+  const urlRegions = [];
+
+  for (let index = 0; index < attributedString.runs.length; index += 1) {
+    const { start, end } = attributedString.runs[index];
```

Comment:
are we getting individual works as the runs? No right?
We need to run isUrl on individual words

Reply:
fixed